bokehjs/src/lib/core/dom.ts

```diff
@@ -145,6 +145,11 @@ export function undisplay(element: HTMLElement): void {
   element.style.display = "none"
 }
 
+export function textOnly(element: HTMLElement): void {
```

Comment:
Maybe this is just my front-end inexperience. Is this the most descriptive name for what this function does? (I don't see the relation)

Reply:
The intention was to make the element dimensions such that any styling on the container itself (in this case, the swatch box) is hidden and so only text will be rendered. This may not be the best implementation. Do you have suggestions for a better approach? I was cautious about making the name specific to swatch, perhaps "hideSwatchBox", as then it likely shouldn't belong in this file. One option is to just remove the function and explicitly set the styling as part of the swatch logic with a comment explaining the purpose.

Reply:
@mattpap will probably have opinions here

Reply:
![image](https://user-images.githubusercontent.com/34218455/100823833-2fbe7a00-3412-11eb-8f8e-02b490b53915.png)


This is what it will look like without this code